### PR TITLE
Check type on assign

### PR DESCRIPTION
**Pull request by Hellblazer**
For issue https://github.com/ChiralBehaviors/Ultrastructure/issues/143

ClassCastException thrown if child is not castable to the required facet type.

:: hehe ::

I have invented a very powerful language pretty cheaply.

### CODE REVIEW

**Comment by Hellblazer** [comment]
not ready for prime time ;)

**Comment by Hellblazer** [comment]
Fixed.  ready for showtime

File: animations/src/main/java/com/chiralbehaviors/CoRE/phantasm/model/PhantasmCRUD.java [hunk 1]
```diff
@@ -212,6 +218,10 @@ public RuleForm addChildren(NetworkAuthorization<RuleForm> facet,
         children.stream()
                 .filter(child -> checkREAD(child,
                                            model.getUnknownNetworkedModel(child)))
+                .filter(child -> {
```
Comment (meleemistress) [inline animations/src/main/java/com/chiralbehaviors/CoRE/phantasm/model/PhantasmCRUD.java]: technically, this should be map(), not filter()

Comment (Hellblazer) [inline animations/src/main/java/com/chiralbehaviors/CoRE/phantasm/model/PhantasmCRUD.java]: It throws an exception, which makes it just an iterator ;)

**Pull request merged**